### PR TITLE
Remove dmspeak

### PR DESCRIPTION
**Pull request by lfdebrux**
Ticket: https://trello.com/c/P8lOZIa6/183-check-whether-weve-missed-anything-from-briefs-frontend

Found some little bits of styles from frontend-toolkit that we no longer want or need.

- Remove uses of single-question-page style
- Remove lead paragraph styling

### CODE REVIEW

File: app/templates/buyers/award.html [hunk 1]
```diff
@@ -39,64 +39,62 @@
 {% block mainContent %}
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <div class="single-question-page">
-      <form method="POST" action="{{ url_for('.award_brief', framework_slug=brief.framework.slug, lot_slug=brief.lotSlug, brief_id=brief.id) }}">
-        <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
+    <form method="POST" action="{{ url_for('.award_brief', framework_slug=brief.framework.slug, lot_slug=brief.lotSlug, brief_id=brief.id) }}">
```
Comment (domoscargin) [inline app/templates/buyers/award.html:42]: Should we add `novalidate` to this form?

File: app/templates/buyers/award_or_cancel_brief.html [hunk 2]
```diff
@@ -39,45 +39,43 @@
...
-      {% else %}
-          <form method="POST" action="{{ url_for('.award_or_cancel_brief', framework_slug=brief.framework.slug, lot_slug=brief.lotSlug, brief_id=brief.id) }}">
-            <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
+      {% if already_awarded %}
+        <h1 class="govuk-heading-l">Requirements already updated for {{ brief.title }}</h1>
+        <p class="govuk-body">
+          <a class="govuk-link" href="{{ url_for('external.get_brief_by_id', framework_family=brief.framework.family, brief_id=brief.id) }}">View the outcome of the requirements</a>
+        </p>
+    {% else %}
+        <form method="POST" action="{{ url_for('.award_or_cancel_brief', framework_slug=brief.framework.slug, lot_slug=brief.lotSlug, brief_id=brief.id) }}">
```
Comment (domoscargin) [inline app/templates/buyers/award_or_cancel_brief.html:48]: Should we add `novalidate` to this form?

File: app/templates/buyers/cancel_brief.html [hunk 3]
```diff
@@ -38,61 +38,59 @@
...
-                </h1>
-              </legend>
-          
-              <p class="govuk-body">
-                  This information will be added to your requirements. Suppliers will receive a link to view the update.
-              </p>
-              <p class="govuk-body">
-                  You’ll still need to contact suppliers to give them feedback.
-              </p>
+      <form method="POST" action="{{ url_for(request.endpoint, framework_slug=brief.framework.slug, lot_slug=brief.lotSlug, brief_id=brief.id) }}">
```
Comment (domoscargin) [inline app/templates/buyers/cancel_brief.html:41]: Should we add `novalidate` to this form?

File: app/templates/buyers/award.html (hunk 1 above)
Comment (lfdebrux) [reply app/templates/buyers/award.html:42]: Good call

Comment (lfdebrux) [reply app/templates/buyers/award.html]: Pushed a commit adding novalidate to this and other forms

**Review by domoscargin — APPROVED** [review]
LGTM!

**Pull request merged**